src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -1256,19 +716,7 @@ export default class ChargingStationService {
     UtilsService.assertIdIsProvided(action, req.body.chargingStationID, MODULE_NAME, 'handleOCPPAction', req.user);
     // Get the Charging station
     const chargingStation = await UtilsService.checkAndGetChargingStationAuthorization(
-      req.tenant, req.user, req.body.chargingStationID, Action.READ, action, null, { withSite: true, withSiteArea: true });
-    // Check auth
-    if (!await Authorizations.canPerformActionOnChargingStation(req.user, command as unknown as Action, chargingStation)) {
-      throw new AppAuthError({
-        ...LoggingHelper.getChargingStationProperties(chargingStation),
-        errorCode: HTTPAuthError.FORBIDDEN,
-        user: req.user,
-        action: command as unknown as Action,
-        entity: Entity.CHARGING_STATION,
-        module: MODULE_NAME, method: 'handleOcppAction',
-        value: chargingStation.id,
-      });
-    }
+      req.tenant, req.user, req.body.chargingStationID, command as unknown as Action, action, null, { withSite: true, withSiteArea: true });
```

Comment:
I would create here an Utils method for mapping the Command enums to Action ones.
If one day I change a value on either enums, this line will not work anymore and no error at compile time.